WARNING 2024-10-25 07:59:07,116 log Not Found: /branch/media/Glico General None_2024-10-22.pdf
WARNING 2024-10-25 08:00:40,737 log Not Found: /branch/media/t_bill_files/Dapaah_Stepehn_2024-10-25.pdf
WARNING 2024-10-25 09:20:38,302 log Not Found: /admin/home
DEBUG 2024-10-25 09:31:21,155 base Exception while resolving variable 'base_color' in template 'socialaccount/login.html'.
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 875, in _resolve_lookup
    current = current[bit]
              ~~~~~~~^^^^^
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'base_color'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 881, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(
                                            ^^^^^^^^
AttributeError: type object 'RequestContext' has no attribute 'base_color'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 891, in _resolve_lookup
    current = current[int(bit)]
                      ^^^^^^^^
ValueError: invalid literal for int() with base 10: 'base_color'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 898, in _resolve_lookup
    raise VariableDoesNotExist(
django.template.base.VariableDoesNotExist: Failed lookup for key [base_color] in [{'True': True, 'False': False, 'None': None}, {'csrf_token': <SimpleLazyObject: 'wUCLuwHyXtZB4DhTq6l954K7uMk3O8gsfjCWtxuCbLcj4cElX9yWTiN8olToRvXA'>, 'request': <WSGIRequest: GET '/accounts/microsoft/login/'>, 'user': <SimpleLazyObject: <function AuthenticationMiddleware.process_request.<locals>.<lambda> at 0x000001A3E6196AC0>>, 'perms': PermWrapper(<SimpleLazyObject: <function AuthenticationMiddleware.process_request.<locals>.<lambda> at 0x000001A3E6196AC0>>), 'messages': <FallbackStorage: request=<WSGIRequest: GET '/accounts/microsoft/login/'>>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'SUCCESS': 25, 'WARNING': 30, 'ERROR': 40}}, {}, {'provider': <allauth.socialaccount.providers.microsoft.provider.MicrosoftGraphProvider object at 0x000001A3E68AF550>, 'process': None}, {'block': <Block Node: content. Contents: [<TextNode: '\n            <div class="'>, <Block Node: whitebox. Contents: []>, <TextNode: '\n                </div>\n '>]>}, {'block': <Block Node: whitebox. Contents: [<TextNode: '\n    <div class="flex ite'>, <IfNode>, <TextNode: '\n        </p>\n    </div>\n'>, <django.template.defaulttags.CsrfTokenNode object at 0x000001A3E68B5010>, <TextNode: '\n        '>, <django.templatetags.i18n.TranslateNode object at 0x000001A3E68B5C10>, <TextNode: '\n        '>, <IncludeNode: template=<FilterExpression '"account/_button.html"'>>, <TextNode: '\n    </form>\n'>]>, 'continue_text': 'Continue'}, {'text': 'Continue'}]
